src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -88,6 +88,11 @@ public void markTask(Task task, Index targetIndex) {
         taskList.setTask(task, targetIndex);
     }
 
+    @Override
+    public void unMarkTask(Task task, Index targetIndex) {
```

Comment:
What if we change the method name to "setUnMarkTask", since we're replacing the Task at the specified Index with the new unmarked Task (and not simply modify the Task at that Index)? "unMarkTask" seems ambiguous and could lead to misconception (initially I thought that the Task parameter was redundant).

Reply:
How about just having `setTask(Task task, Index targetIndex)` ? Meaning that anything that replace old Task object with a new Task object will utilize this `setTask` method without creating duplicate methods for edit mark unmark?

Reply:
Will this be more intuitive for you? This is because lets say in the UnMarkTaskCommand, 
we are setting a new Task (as unmarked) to replace the targetIndex Task. 
Meaning that `model.unMarkTask(Task)` will be changed to `setTask(Task)`.
![image](https://user-images.githubusercontent.com/56034480/194739989-0284ed62-dd36-4824-85ba-fe2cb7b6a568.png)